crates/bevy_render/src/camera/camera.rs

```diff
@@ -219,6 +223,15 @@ impl Camera {
     ///
     /// To get the coordinates in Normalized Device Coordinates, you should use
     /// [`world_to_ndc`](Self::world_to_ndc).
+    ///
+    /// Returns None if any of these conditions occur:
```

Comment:
```suggestion
    /// Returns `None` if any of these conditions occur:
```

(And same change elsewhere)

Reply:
Fixed